pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -206,6 +297,17 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         action: () => subvolume_create(volume, subvol, (mounted && !opt_ro) ? mount_point : mount_point_in_parent),
     });
 
+    let delete_excuse = create_excuse;
```

Comment:
I think this is not correct. Deletion needs the parent to be mounted rw, mounting the subvol-to-be-deleted but not the parent will not work. Or does it?

Reply:
No it doesn't. When mount_point_in_parent is `null`, Cockpit tries to delete the "null" subvolume.